pkgs/development/compilers/cudatoolkit/versions.toml

```diff
@@ -61,7 +61,13 @@ sha256 = "sha256-qyGa/OALdCABEyaYZvv/derQN7z8I1UagzjCaEyYTX4="
 gcc = "gcc11"
 
 ["11.7"]
-version = "11.7.0"
-url = "https://developer.download.nvidia.com/compute/cuda/11.7.0/local_installers/cuda_11.7.0_515.43.04_linux.run"
-sha256 = "sha256-CH/fy7ofeVQ7H3jkOo39rF9tskLQQt3oIOFtwYWJLyY="
+version = "11.7.1"
+url = "https://developer.download.nvidia.com/compute/cuda/11.7.1/local_installers/cuda_11.7.1_515.65.01_linux.run"
+sha256 = "sha256-UihqKXBlSbfQ/usOfj7KGxUofENqafqICtOFsb4+BNs="
+gcc = "gcc11"
```

Comment:
Why bump 11.7.0 -> 11.7.1? This will cause a discrepancy between the `cudatoolkit` and redistrib package set versions which is a future footgun.

I'm in favor of jointly updating the `cudatoolkit` and redistrib 11.7.x versions in a separate PR if at all possible.

Reply:
done.

Reply:
i'm confused... it looks like the 11.7.0 -> 11.7.1 bump is still in this PR?

Reply:
sorry about that, I forgot to remove the patch when rebasing.